### ISSUE TITLE
Override return types of hooks

### ISSUE THREAD

**Issue by nopitown** [7nohe/openapi-react-query-codegen#29]
Currently, when the response type schema is not present for endpoints(Open API), the return type of the hooks is "any". It would be nice if we were able to override them.

Example:
```
const { data } = useFoo<MyType>();
```

PS: Thank you for maintaining this library! I use it in our day-to-day, and it has saved me a ton of hours.

**Comment by 7nohe** [comment]
Hi @nopitown,  
I'm very pleased to hear that you use my library and that it's been helpful for your development. Thank you for your feedback and for your support through GitHub Sponsors!

I've just released v0.5.1. Now you can override the return type and errors.

```ts
const { data, error } = useFoo<[], MyType, MyError>();
```

**Comment by seriouslag** [comment]
An update:

In v0.5.2 we have changed the types, no longer using Omit and changed the order of the Type parameters.

In the following example, data's type will be a union of undefined and MyType.
```typescript
const { data, error } = useFoo<MyType, MyError>();
```
